modules/introductions/infrastructure/preview/preview-email-gateway.ts

```
@@ -0,0 +1,72 @@
+
+			deps.logger.info({
+				event: "demo.email.preview_captured",
+				details: {
+					previewId,
+					subject: params.subject,
+					recipientCount: lastPreview.to.length,
+				},
+			});
+
+			return { id: asEmailId(previewId) };
+		},
```

Biome (2.4.4)

[error] This async function lacks an await expression.

(lint/suspicious/useAwait)